docs/GDI-SOP_sop-accessioning.md

```diff
@@ -109,13 +109,13 @@ The repository is divided in two main directories:
   - `docs/`: Contains all other supporting documents.
 
 ## SOP Life-cycle
```

Comment:
We have described very well how requests for new SOPs will be handled, but not (as I can see) how a request for changing an existing SOP template or SOP will be handled. I'm not saying that we should spend time on that now, but I think it might be good to be aware of.

Reply:
Definitely, I had these in mind, but preferred to focus on the request of SOPs, as there are none to request changes on.

I would expect a similar SOP, or a modification of the SOP I wrote in the other PR, to explain in detail that whole process.